Geometry/CMSCommonData/python/cmsExtendedGeometry2021XML_cfi.py

```diff
@@ -242,6 +242,7 @@
         'Geometry/MuonCommonData/data/gem11/TDR_BaseLine/gem11.xml',
         'Geometry/MuonCommonData/data/csc/2021/v2/csc.xml',
         'Geometry/MuonCommonData/data/mfshield/2017/v2/mfshield.xml',
+        'Geometry/VeryForwardData/data/2021/v1/pps.xml',
```

Comment:
@civanch - this file should be `Geometry/VeryForwardData/data/pps/2021/v1/pps.xml`

Reply:
@ianna , you mean, that VeryForwardData should be reorganized? In that case too many renames will be needed.

Reply:
@civanch - yes, agree. However, this particular line is a new inclusion in the scenario, so it should follow the CMS convention IMHO